records_mover/records/targets/table/move_from_records_directory.py

```diff
@@ -50,7 +50,10 @@ def load_plan(self) -> RecordsLoadPlan:
 
     def load(self, driver: DBDriver) -> Optional[int]:
         plan = self.load_plan
-        return driver.load(schema=self.tbl.schema_name, table=self.tbl.table_name,
+        loader = driver.loader_from_records_directory()
+        # TODO: Can this take in a loader so we don't have to do this assertion here?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker